minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -85,9 +85,13 @@ java.arg.4=-Djava.net.preferIPv4Stack=true
 java.arg.5=-Dsun.net.http.allowRestrictedHeaders=true
 java.arg.6=-Djava.protocol.handler.pkgs=sun.net.www.protocol
 
+# Sets the provider of SecureRandom to /dev/urandom to prevent blocking on VMs
+java.arg.7=-Djava.security.egd=file:/dev/urandom
```

Comment:
@trixpan have you tested that this works in Java 8?

I've used a slightly modified version found [on stack overflow](http://stackoverflow.com/questions/137212/how-to-solve-performance-problem-with-java-securerandom#answer-2325109):
```
-Djava.security.egd=file:/dev/./urandom
```